### ISSUE TITLE
A11y : Cannot focus a panel via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46487]
**Steps to reproduce**:

In a dashboard, a keyboard user cannot access a panel (edit/view...) when it has no content or is a certain type (so far I noticed Gauge, Stat and Logs can't be focussed)

See Also:

[This design doc
](https://docs.google.com/document/d/1Nc_yBpzemBTqRRWvQPpTk3cwAZvW33ZqXRkEmVzKR54/edit#heading=h.5sybau7waq2q
)

[This Demo Video
](https://drive.google.com/file/d/1jUabAOxavTjVddUy5v1EeC_t1IIgbHyI/view)

**Actual Result**:

There’s no way to focus a panel, only its content (and not all types).

**Expected Result**

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by axelavargas** [comment]
This issue is blocked by https://github.com/grafana/grafana/issues/31080

**Comment by natellium** [comment]
This seems to be fixed with the new panel header
